clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -165,6 +165,10 @@
     /** <code>connections.max.idle.ms</code> */
     public static final String CONNECTIONS_MAX_IDLE_MS_CONFIG = CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_CONFIG;
 
+    /** <code>pattern.subscription.interval.ms</> **/
+    public static final String PATTERN_SUBSCRIPTION_INTERVAL_MS_CONFIG = "pattern.subscription.interval.ms";
```

Comment:
Do we need a new config for subscription interval? Could we piggy-back the logic on regular metadata refresh?